### PR TITLE
Prettier to basics folder

### PR DESCRIPTION
**Pull request by CodinCat**
As discussed in #2385 , I'm splitting that PR to smaller ones.

This PR apply prettier to docs/basics folder.

### CODE REVIEW

File: docs/basics/ExampleTodoList.md [hunk 1]
```diff
@@ -83,14 +83,9 @@ const todo = (state = {}, action) => {
...
-      return [
-        ...state,
-        todo(undefined, action)
-      ]
+      return [...state, todo(undefined, action)]
     case 'TOGGLE_TODO':
-      return state.map(t =>
-        todo(t, action)
-      )
+      return state.map(t => todo(t, action))
```
Comment (timdorr) [inline docs/basics/ExampleTodoList.md]: So, these are some examples of style violations that actually serve as a documentation aid. The multiple lines here let us break things down into smaller steps, which is helpful when you're reading through documentation and want to know exactly what's going on. I'd leave this as-is.

File: docs/basics/ExampleTodoList.md [hunk 2]
```diff
@@ -328,10 +319,7 @@ const mapDispatchToProps = (dispatch, ownProps) => {
   }
 }
 
-const FilterLink = connect(
-  mapStateToProps,
-  mapDispatchToProps
-)(Link)
+const FilterLink = connect(mapStateToProps, mapDispatchToProps)(Link)
```
Comment (timdorr) [inline docs/basics/ExampleTodoList.md]: Same thing here. In lieu of a decorator, the multiple lines for `connect` calls are easier to read through.

**Comment by timdorr** [comment]
@CodinCat I didn't mean just those specific lines to be fixed. I meant all cases in the code like that. There are still several ones left over.

**Comment by CodinCat** [comment]
Oops didn't notice that. Updated.

**Comment by timdorr** [comment]
BTW, the same applies to the other PRs too. Didn't want to give you like 19280205937 comments to sift through.

**Comment by CodinCat** [comment]
Yeah I'm working on other PRs

**Comment by CodinCat** [comment]
All other PRs are completed.

File: docs/basics/UsageWithReact.md [hunk 3]
```diff
@@ -321,10 +320,7 @@ const mapDispatchToProps = (dispatch, ownProps) => {
   }
 }
 
-const FilterLink = connect(
-  mapStateToProps,
-  mapDispatchToProps
-)(Link)
+const FilterLink = connect(mapStateToProps, mapDispatchToProps)(Link)
```
Comment (timdorr) [inline docs/basics/UsageWithReact.md]: Looks like this one still made it through.

File: docs/basics/ExampleTodoList.md [hunk 4]
```diff
@@ -83,10 +83,7 @@ const todo = (state = {}, action) => {
 const todos = (state = [], action) => {
   switch (action.type) {
     case 'ADD_TODO':
-      return [
-        ...state,
-        todo(undefined, action)
-      ]
+      return [...state, todo(undefined, action)]
```
Comment (timdorr) [inline docs/basics/ExampleTodoList.md]: Can you revert this too? I think it's a little easier to read when multi-lined.

**Comment by timdorr** [comment]
Oh good, you had maintainer editing on. Saved us some cycle time. 

LGTM!

**Pull request merged**